apps/server/test/modules/learnroom/course.e2e-spec.ts

```diff
@@ -3,17 +3,20 @@ import { ExecutionContext, INestApplication } from '@nestjs/common';
 import request from 'supertest';
 import { Request } from 'express';
 import { MikroORM } from '@mikro-orm/core';
-import { EntityManager } from '@mikro-orm/mongodb';
+import { EntityManager, ObjectId } from '@mikro-orm/mongodb';
 import { ServerModule } from '@src/server.module';
 import { CourseMetadataListResponse } from '@src/modules/learnroom/controller/dto';
 import { JwtAuthGuard } from '@src/modules/authentication/guard/jwt-auth.guard';
-import { userFactory, courseFactory, cleanUpCollections, createCurrentTestUser } from '@shared/testing';
+import { userFactory, courseFactory, cleanUpCollections, roleFactory } from '@shared/testing';
 import { ICurrentUser, User } from '@shared/domain';
 
-const modifyCurrentUserId = (currentUser: ICurrentUser, user: User) => {
-	currentUser.user.id = user.id;
-	currentUser.userId = user.id;
-};
+const mapToCurrentUser = (user: User) =>
+	({
+		userId: user.id,
+		roles: user.roles.getItems().map((r) => r.id),
+		schoolId: user.school.id,
+		accountId: new ObjectId().toHexString(),
+	} as ICurrentUser);
```

Comment:
**code duplication**: can this methode into test-utils?

Reply:
yes. we could move that to `shared/testing`.